modules/markdown/markdown.go

```diff
@@ -53,10 +53,11 @@ func isLink(link []byte) bool {
 // IsMarkdownFile reports whether name looks like a Markdown file
 // based on its extension.
 func IsMarkdownFile(name string) bool {
-	name = strings.ToLower(name)
```

Comment:
This now uses the markdown file extensions set in app.ini. This is so you can assume .txt or files with no extensions, for example, can be rendered as Markdown on a personal basis.